bom/build.gradle.kts

```diff
@@ -58,6 +58,7 @@ dependencies {
         apiv("org.locationtech.jts:jts-core")
         apiv("org.locationtech.jts.io:jts-io-common")
         apiv("org.locationtech.proj4j:proj4j")
+        apiv("org.locationtech.proj4j:proj4j-epsg")
```

Comment:
If proj4j and proj4j-epsg are likely to have the same version number there's no need for this line, or the line in gradle.properties.